rpyc/rpyc_slave/FileMonitorService.py

```diff
@@ -0,0 +1,33 @@
+import rpyc
+import os
+import time
+from threading import Thread
+
+
+# The fact that we inherit from rpyc.SlaveService makes the exposed_ prefix not to work.
```

Comment:
Services supposed to expose some methods. Like private and public, and this is done using the exposed prefix of the method's name.
But I realize I've been stupid, because FileMonitor is just a regular class (FileMonitorService is the rpyc service).